addons/ofxAndroid/ofAndroidLib/src/cc/openframeworks/OFAndroid.java

```diff
@@ -54,6 +58,105 @@ private static String getPackageName(){
 		return OFAndroidLifeCycle.getActivity().getPackageName();
 	}
 
+	public static String parsePermission(int permission){
+		String PERMISSION = "";
+		switch(permission){
+			case 0:
```

Comment:
having just numbers here looks very error prone can't we just get this value through jni's getStaticField(ID) instead of having the switch case here?